internal/kafka/internal/presenters/kafka.go

```diff
@@ -114,11 +115,16 @@ func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest, kafkaConfig *config.K
 	}, nil
 }
 
-func setBootstrapServerHost(bootstrapServerHost string) string {
-	if bootstrapServerHost != "" {
-		return fmt.Sprintf("%s:443", bootstrapServerHost)
+func setBootstrapServerHost(request *dbapi.KafkaRequest) string {
+	if request.BootstrapServerHost != "" {
+		switch request.Status {
+		case constants.KafkaRequestStatusSuspended.String(), constants.KafkaRequestStatusSuspending.String(), constants.KafkaRequestStatusResuming.String():
+			return ""
```

Comment:
I am wondering if we should also update the description: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/f9ab2b6af9c72e956df5e1833a3a2235a1b982aa/openapi/kas-fleet-manager.yaml#L981 ? 
@JameelB

Reply:
same for the admin endpoints api